CONTRIBUTING.md

```diff
@@ -8,10 +8,10 @@ of this guide and also our [code of conduct](CODE_OF_CONDUCT.md).
 :star2: **Third** - the best way to get started contributing, is to use the
 tool in anger and then to submit bugs and features through GitHub.
 In particular, in helping to develop the parser, examples of queries
-which don't parse as expected are especially helpful.
+that do not parse as expected are especially helpful.
 
 :star2: **Fourth** - making sure that our documentation is up-to-date and useful
-for new users is really important. If you're a new user, you're in precisely
+for new users is important. If you are a new user, you are in precisely
```

Comment:
I think this is a case where conciseness "improvement" is perhaps too far. I'd suggest adding the "really" back in as documentation is really, really, really important.

```suggestion
for new users is really important. If you are a new user, you are in precisely
```